src/pages/index.js

```diff
@@ -32,53 +155,43 @@ const addModalFormValidator = new FormValidator(
   variables.addModalForm
 );
 addModalFormValidator.enableValidation();
-//------------------------------------------------------------------------------
-const editFormPopup = new PopupWithForm("#profile-edit-modal", (formData) => {
-  user.setUserInfo({
-    name: formData.title,
-    job: formData.description,
-  });
-});
-
-editFormPopup.setEventListeners();
 
-const user = new UserInfo({
-  nameSelector: "#profile-info-title",
-  jobSelector: "#profile-info-description",
-});
-
-variables.profileEditButton.addEventListener("click", () => {
-  const userInfo = user.getUserInfo();
-  variables.profileModalNameInput.value = userInfo.name.trim();
-  variables.profileModalDescriptionInput.value = userInfo.job.trim();
-  editFormPopup.open();
-});
+const avatarFormValidator = new FormValidator(
+  formConfig,
+  variables.avatarModalForm
+);
+avatarFormValidator.enableValidation();
 
-const addFormPopup = new PopupWithForm(
+const addCardPopup = new PopupWithForm(
   "#profile-add-modal",
   ({ title: name, link }) => {
-    renderCard({ name, link });
+    createCardApi({ name, link });
     variables.addModalForm.reset();
     addModalFormValidator.disableSubmitButton();
```

Comment:
You need to **reset** the form and disable the button only when you **open** it or better in block `then` not to do that immediately after `submit` because when there is an error the popup will still be opened and the user can try to send the data again to the server without having to type the data into the inputs once more